iXBRLViewerPlugin/iXBRLViewer.py

```diff
@@ -403,6 +411,33 @@ def createViewer(self, scriptUrl=DEFAULT_VIEWER_PATH, useStubViewer=False, showV
             iv.addFilingDoc(filingDocZipPath)
             self.taxonomyData["filingDocuments"] = filingDocZipName
 
+        localDocs = defaultdict(set)
+        for path, doc in dts.urlDocs.items():
+            if isHttpUrl(path):
+                continue
+            if doc.type == Type.INLINEXBRL:
+                localDocs[doc.basename].add('inline')
+            elif doc.type == Type.INLINEXBRLDOCUMENTSET:
+                localDocs[doc.basename].add('docset')
```

Comment:
I'm not familiar enough with Arelle.  What does `docset` vs `inline` mean?  It seems like they should just be the same for ixbrl-viewer?

Reply:
You're right. The distinction from within the viewer is unimportant. I've combined them.